homeassistant/components/hassio/__init__.py

```diff
@@ -183,6 +185,13 @@ class APIEndpointSettings(NamedTuple):
 }
 
 
+@dataclass
+class HassioServiceInfo(BaseServiceInfo):
```

Comment:
Alternatively we can keep the definition in discovery and import an unused import here.

Reply:
Do I need to rollback the move?

Reply:
I couldn't resolve the circular import, so I have moved it back to discovery, and imported it as an unused import.